managing-os/pentesting.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Penetration Testing
+permalink: /doc/pentesting/
+---
+
+**Legal notice:**
+
+The usage of penetration testing tools outside your own laboratory environment requires the permission of the organization you attack. Penetration testing without such a permission can have legal consequences.
+
+To avoid such legal conflicts please refer to the [Code of Ethics](https://www.eccouncil.org/Support/code-of-ethics).
+
+Penetration Testing
+===================
+
+"A penetration test, informally pen test, is an attack on a computer system that looks for security weaknesses, potentially gaining access to the computer's features and data." (source [Penetration test](https://en.wikipedia.org/wiki/Penetration_test)).
+
+Penetration Testing Distributions:
+----------------------------------
+
+The following install instructions explain how to setup a penetration testing distribution within Qubes OS.  
+
+- [BlackArch](/doc/blackarch/)
+- [Kali](/doc/kali/)
+- [PenTester Framework (PTF)](/doc/ptf/)
```

Comment:
But this isn't about "2 levels only ToC" at all. It's just about keeping uniformity with existing docs. The way the Whonix docs are laid out, the specific pages aren't linked from the main ToC.